### ISSUE TITLE
Prevent Override of [Authorize]

### ISSUE THREAD

**Issue by KirschnerMatt** [Jasig/dotnet-cas-client#42]
I'm trying to use the .Net Cas Client alongside my custom Asp.Net Membership implementation. To do this, I would like to bind the client's Cas authorization to it's own Data Annotation, rather than have it simply override the default.

Is this possible?

**Comment by hokiecoder** [comment]
@KirschnerMatt are you still experiencing this issue/have this need? I am trying to help work on the open issues for the project, so if you still want this answered I'll try to help look into it.

**Comment by KirschnerMatt** [comment]
Nope, I wound up rolling my own custom Cas client using MVC filters.

**Comment by hokiecoder** [comment]
@KirschnerMatt Thanks for the quick reply back. Glad to hear you worked out a solution.

@serac OK to close this (now) non-issue?

**Comment by serac** [comment]
Actually I think this sounds like a useful feature. Let's leave open and consider how we might accomplish it, possibly in context of a framework refactoring.

**Comment by hokiecoder** [comment]
Sounds good. I'll add it to my to-do list. Any feedback/suggestions from other contributors on how this might best be accomplished will be greatly appreciated.

**Comment by phantomtypist** [comment]
We've done something similar with our applications however I'm not sure how easy it would be to do here and now.  Let's say we introduced an action attribute called [CasAuthorize] that people could use, we'd have to come up with a way to not break people's existing code (i.e. they are already using [Authorize].)

Again, we'd need to take a look at the do-ability of this one without making too big of a breaking change for people, right?

Worst case we'd do it in a 2.x release to indicate major breaking change.

@scottt732 thoughts?
